README.md

```diff
@@ -1,2 +1,19 @@
 # OHSNAP-SWFirewall
+
 Distributed firewall software used in the OHSNAP project. Can run independently of OHSNAP hardware.
+
+## Folder Structure
```

Comment:
Would it make sense to have this section? I think the primary utility here is to help newcomers and outsiders understand the repo at a broad level.

On the other hand, this is duplicating information and will be out of date unless regularly sync'd with the actual folder hierarchy.

Reply:
I think it’s fine to leave it in